conductor/Cargo.toml

```diff
@@ -14,28 +14,27 @@ chrono = { version = "0.4.24", features = ["serde"] }
 controller = { path = "../tembo-operator", package = "controller" }
 env_logger = "0.10.0"
 futures = "0.3.28"
-k8s-openapi = { version = "0.18.0", features = ["v1_25", "schemars"] }
+k8s-openapi = { version = "0.23.0", features = ["v1_29", "schemars"] }
```

Comment:
I think since we're updating the version here we'll also have to update the version in our control-plane? I'm not 100% sure but we'll find out next time we need to build it.